legacy_jetstream.md

```diff
@@ -0,0 +1,712 @@
+# JetStream
+
+JetStream is the NATS persistence engine providing streaming, message, and
+worker queues with At-Least-Once semantics. JetStream stores messages in
+_streams_. A stream defines how messages are stored and how long they persist.
+To store a message in JetStream, you simply need to publish to a subject that is
+associated with a stream.
+
+Messages are replayed from a stream by _consumers_. A consumer configuration
+specifies which messages should be presented. For example a consumer may only be
+interested in viewing messages from a specific sequence or since a specific
+time. The configuration also specifies if the server should require for a
+message to be acknowledged, and how long to wait for acknowledgements. The
+consumer configuration also specifies options to control the rate at which
+messages are presented to the client.
+
+For more information about JetStream, please visit the
+[JetStream repo](https://github.com/nats-io/jetstream).
+
+## JetStream client for JavaScript
+
+The new generation of Javascript clients:
+
+- [nats.js](https://github.com/nats-io/nats.js)
+- [nats.ws](https://github.com/nats-io/nats.ws)
+- [nats.deno](https://github.com/nats-io/nats.deno)
+
+all support JetStream. Please report any issues you find.
+
+## JetStreamManager
+
+The client provides CRUD functionality to manage streams and consumers, via
```

Comment:
```suggestion
The client provides CRUD functionality to manage streams and consumers via
```

Reply:
Ah these are showing new but they are actually the old readme which I am reworking for simplification in a different pr